Library/Formula/postgresql.rb

```diff
@@ -77,8 +82,16 @@ def install
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    system "pkill postgres || true"
+
+    if (var/"postgres").exist?
+      mv var/"postgres", var/"postgres-#{version_suffix}"
```

Comment:
We've always had to launch ` postgres ` manually, as such. It never knows where its ` var ` directory is without a nudge. We handle this with the ` plist ` statement printed after install:

```
plist_options :manual => "postgres -D #{HOMEBREW_PREFIX}/var/postgres"
```

which we've just changed to:

```
plist_options :manual => "postgres -D #{HOMEBREW_PREFIX}/var/postgres-9.4"
```

And the applicable mentions in the ` plist ` file itself have been changed as well. It's just a case of a user doing the usual ` ln_sf prefix/blah.plist ~/Library/LaunchAgents && launchctl unload blah && launchctl load blah ` dance, and then ` postgres ` is golden.